build/Build.Steps.Windows.cs

```diff
@@ -25,15 +25,17 @@ partial class Build
                 ? new[] { MSBuildTargetPlatform.x64, MSBuildTargetPlatform.x86 }
                 : new[] { MSBuildTargetPlatform.x86 };
 
-            // Can't use dotnet msbuild, as needs to use the VS version of MSBuild
-            MSBuild(s => s
-                .SetTargetPath(MsBuildProject)
-                .SetConfiguration(BuildConfiguration)
-                .SetTargets("BuildCpp")
-                .DisableRestore()
-                .SetMaxCpuCount(null)
-                .CombineWith(platforms, (m, platform) => m
-                    .SetTargetPlatform(platform)));
+            foreach (var project in Solution.GetNativeSrcProjects())
+            {
+                // Can't use dotnet msbuild, as needs to use the VS version of MSBuild
+                MSBuild(s => s
+                    .SetTargetPath(project)
+                    .SetConfiguration(BuildConfiguration)
+                    .DisableRestore()
+                    .SetMaxCpuCount(null)
```

Comment:
No from your change, but, I am curious: isn't the default good enough? It seems strange to call it and pass `null`.

Reply:
> If you don't include this switch, the default value is 1. If you include this switch without specifying a value, MSBuild will use up to the number of processors in the computer.

Seems this api has quite poor readability, but null value here makes sense.